### PR TITLE
fix pipeline spec since blank macros are no longer allowed

### PR DESCRIPTION
**Pull request by seanfeld**
This pr broke our integration tests https://github.com/cdapio/cdap/commit/f3e93f53eb32f79f2a1dbc62d9667782748234a4#diff-51bd0306753b1006d73ab2c7aa60ea66a1606b0f69f1eb79e31981c913afbc8e
with new behavior. This pr fixes the test to adjust to that behavior.

### CODE REVIEW

File: cypress/integration/pipeline.spec.ts [hunk 1]
```diff
@@ -126,9 +127,9 @@ describe('Creating a pipeline', () => {
 
     cy.contains('error found').should('exist');
 
-    // Fix missing field to resolve error
-    cy.get('[data-cy="message"]').within(() => {
-      cy.get('button').click();
+    cy.get('[data-cy="message"] textarea').type(TEST_EMAIL_MACRO, {
```
Comment (njbriggs) [inline cypress/integration/pipeline.spec.ts:130]: Nit: the comment is still valid. The test triggers an empty field error before validating correctly

**Review by njbriggs — COMMENTED** [review]
Do any other tests do this? Cypress dashboard shows 4 failures.

**Comment by seanfeld** [comment]
@njbriggs I'll check on the other ones - i may be all the same.

**Comment by seanfeld** [comment]
@njbriggs all the pipeline.spec tests pass correctly

**Review by njbriggs — APPROVED** [review]

File: package.json [hunk 2]
```diff
@@ -34,7 +34,7 @@
     "precypress-open": "node cypress/pre-startup.js",
     "precypress": "node cypress/pre-startup.js",
     "cypress": "cypress run",
-    "cypress-open": "cypress open",
+    "cypress-open": "node -e \"console.log('**DISABLE BEYOND CORP EXTENSION OR THIS WONT WORK**')\" && cypress open",
```
Comment (mrahanjam) [inline package.json:37]: 👍

**Review by mrahanjam — APPROVED** [review]
LGTM

**Pull request merged**